components/classes/fsi/policy-details.schema.json

```diff
@@ -5,11 +5,14 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/xdm/datatypes/policy-details",
+  "$id": "https://ns.adobe.com/xdm/classes/fsi/policy-details",
```

Comment:
Can we just name it as "policy" to be consistent with other classes ?

Reply:
Yes.  I just updated it accordingly.